Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -340,10 +341,10 @@ private void EjectBeaker(ReagentGrinderComponent component, IEntity? user)
                             }
 
                             if (component.HeldBeaker.CurrentVolume +
-                                juiceMe.ResultSolution.TotalVolume * juiceEvent.Scalar >
+                                juiceMe.JuiceSolution!.TotalVolume * juiceEvent.Scalar >
```

Comment:
no `!`, properly handle it

Reply:
Does it need it? Fact you're raising an event means, that prototype has that value. I'm not sure what the correct behavior if you raised an event on a prototype that had its field changed.